backend/src/app.module.ts

```diff
@@ -0,0 +1,20 @@
+import { Module } from '@nestjs/common';
+import { SequelizeModule } from '@nestjs/sequelize';
+import { usersModule } from './users/users.module';
+
+
+@Module({
+  imports: [SequelizeModule.forRoot({
+    dialect: 'postgres',
+    host: 'localhost',
+    port: 5438,
+    username: 'postgres',
+    password: 'postgres',
+    database: 'pizzaapp',
```

Comment:
```suggestion
  imports: [SequelizeModule.forRoot({
    dialect: 'postgres',
    host: 'localhost',
    port: 5438,
    username: 'postgres',
    password: 'postgres',
    database: 'pizzaapp',
```
Use all important things in .env